cupy/__init__.py

```diff
@@ -1146,8 +1146,21 @@ def _embed_signatures(dirs):
 
 
 _embed_signatures(globals())
-_embed_signatures(fft.__dict__)
-_embed_signatures(linalg.__dict__)
-_embed_signatures(random.__dict__)
-_embed_signatures(sparse.__dict__)
-_embed_signatures(testing.__dict__)
```

Comment:
I removed these because they should be automatically handled by the `globals()` call above.